verification/extendTCI.go

```diff
@@ -50,10 +51,108 @@ func TestExtendTCI(d TestDPEInstance, c DPEClient, t *testing.T) {
 	}
 
 	// Cross-check current and cumulative measurement by CertifyKey
-	verifyMeasurementsByCertifyKey(c, t, handle, defaultTci, tciValue, hasher)
+	verifyMeasurementsByCertifyKey(c, t, handle, defaultTci, tciValue, false, hasher)
 }
 
-func verifyMeasurementsByCertifyKey(c DPEClient, t *testing.T, handle *ContextHandle, label []byte, tciValue []byte, hasher hash.Hash) {
+// Check whether the ExtendTCI command with derived child context.
+func TestExtendTciOnDerivedContexts(d TestDPEInstance, c DPEClient, t *testing.T) {
+	var err error
```

Comment:
This tests ExtendTCI on derived context using CertifyKey instead of GettaggedTCI (which was removed in PR 257).

Reply:
Please restrict this PR to only sign tests

Reply:
Removed the other changes, synced this branch with latest as it had conflicts.